website/project/model.py

```diff
@@ -3126,14 +3161,29 @@ def to_json(self):
 
 
 class Sanction(StoredObject):
-    """Sanction object is a generic way to track approval states"""
-
+    """Sanction class is a generic way to track approval states"""
+    # Tell modularodm not to attach backends
     abstract = True
 
+    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+
+    # Neither approved not cancelled
     UNAPPROVED = 'unapproved'
+    # Has approval
     APPROVED = 'approved'
+    # Rejected by at least one person
     REJECTED = 'rejected'
 
+    state = fields.StringField(
```

Comment:
The `state` field is defined twice: once on line 3177, and again on line 3220. Please consolidate these (keeping validation etc as appropriate)